prac_06/programming_language.py

```diff
@@ -0,0 +1,27 @@
+class ProgrammingLanguage:
+    """Programming language information"""
+
+    def __init__(self, name, typing, reflection, year):
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+        self.name = name
+
+    def is_dynamic(self):
+        return self.typing == "Dynamic"
+
+    def __str__(self):
+        return f"{self.name}, {self.typing} Typing, Reflection={self.reflection}, First appeared in {self.year}"
+
```

Comment:
I'm not sure of this myself, but are the double underscore methods meant to go first? Then the rest of the methods follow underneath?